docs/modelserving/v1beta1/llm/huggingface/README.md

```diff
@@ -1,17 +1,18 @@
 # Deploy the Llama2 model with Hugging Face LLM Serving Runtime
-The Hugging Face LLM serving runtime implements a runtime that can serve Hugging Face LLM model out of the box. 
+The Hugging Face LLM serving runtime implements a runtime that can serve Hugging Face LLM model out of the box.
```

Comment:
Let's move to the separate tutorial for huggingface transformer as here is focusing on LLM